apps/ios/Shared/Views/Onboarding/SimpleXInfo.swift

```diff
@@ -39,13 +39,14 @@ struct SimpleXInfo: View {
                     Spacer()
 
                     VStack(alignment: .leading) {
-                        infoRow("privacy", "Privacy redefined",
-                                "The 1st platform without any user identifiers – private by design.", width: 48)
-                        infoRow("shield", "Immune to spam and abuse",
-                                "People can connect to you only via the links you share.", width: 46)
-                        infoRow(colorScheme == .light ? "decentralized" : "decentralized-light", "Decentralized",
-                                "Open-source protocol and code – anybody can run the servers.", width: 44)
+                        onboardingInfoRow("privacy", "Privacy redefined",
+                                "No user identifiers.", width: 48)
+                        onboardingInfoRow("shield", "Immune to spam",
+                                "You decide who can connect.", width: 46)
```

Comment:
Or "You decide who to connect with"
```suggestion
                                "You decide who can connect to you.", width: 46)
```